src/main/scala/cromwell/server/WorkflowManagerSystem.scala

```diff
@@ -0,0 +1,15 @@
+package cromwell.server
+
+import akka.actor.ActorSystem
+import cromwell.engine.WorkflowManagerActor
+
+trait WorkflowManagerSystem {
+  val systemName = "cromwell-system"
+  val actorSystem = ActorSystem(systemName)
+
+  actorSystem.registerOnTermination({
+    actorSystem.log.info(s"$systemName shutting down")
+  })
```

Comment:
Heh :smiling_imp: 

Pretty sure this can be one-lined, but I'm a lot less sure that's a good idea.  The curlies make it clear this is a call by name, so maybe this should stay the way you have it?